spec/components/contributor_email_settings_spec.rb

```diff
@@ -5,20 +5,22 @@
 RSpec.describe ContributorEmailSettings::ContributorEmailSettings, type: :component do
   subject { render_inline(described_class.new(**params)) }
 
+  let(:organization) { create(:organization) }
   let(:contributor) do
     create(:contributor,
            email: 'muster@example.org',
            first_name: 'Max',
            last_name: 'Mustermann',
-           created_at: '2021-01-01')
+           created_at: '2021-01-01',
+           organization: organization)
   end
 
-  let(:params) { { contributor: contributor } }
+  let(:params) { { contributor: contributor, organization: organization } }
 
   it { should have_css('h2', text: 'E-Mail') }
   it { should have_css('p', text: 'Max Mustermann hat sich via E-Mail angemeldet.') }
 
-  it { should have_css("form[action='/contributors/#{contributor.id}']") }
+  it { should have_css("form[action='/#{contributor.organization_id}/contributors/#{contributor.id}']") }
```

Comment:
I prefer to set hard-coded ids and repeat them in the assertions.

Reply:
you're right on this as it won't work when a n:n is introduced for contributors and organizations. changed with 9ef1a28cee914e2892effb458dfbd97ee0a48702

Reply:
I think that is not what @roschaefer was saying. I know from working with Robert that he prefers to set the id to some hard-coded value `234_321` and then assert this hard-coded value over using something like `organization.id`, which can give false positives.